RuleDocumentation/UseConsistentWhitespace.md

```diff
@@ -53,7 +54,11 @@ Checks if a comma or a semicolon is followed by a space. E.g. `@(1, 2, 3)` or `@
 
 #### CheckPipe: bool (Default value is `$true`)
 
-Checks if a pipe is surrounded on both sides by a space. E.g. `foo | bar` instead of `foo|bar`.
+Checks if a pipe is surrounded on both sides by a space but ignores redundant whitespace. E.g. `foo | bar` instead of `foo|bar`.
+
+#### CheckPipeForRedundantWhitespace : bool (Default value is `$false`)
+
+Checks if a pipe is surrounded by redundant whitespace (i.e. more than 1 whitespace). E.g. `foo | bar` instead of `foo|bar`.
```

Comment:
Is the example here accurate? Or is it something like:

```powershell
foo    | bar
```

vs

```powershell
foo | bar
```

(GitHub seems to normalise spaces in inline code...)

Reply:
Yes, you're right, I forgot to update the example as well.